packages/server/src/routes/grants.js

```diff
@@ -44,6 +44,8 @@ router.get('/', requireUser, async (req, res) => {
             positiveInterest: req.query.positiveInterest ? true : null,
             rejected: req.query.rejected ? true : null,
         },
+        orderBy: req.query.orderBy,
+        ascending: req.query.ascending,
```

Comment:
fwiw, this wasn't really necessary because the `...req.query` above already passes these fields.

Reply:
At some point during development, I didn't see them in the debugger, so I force-added them. In the [front-end change list](https://github.com/usdigitalresponse/usdr-gost/tree/425-sort-grants-frontend) I actually renamed one to keep naming consistent across all the layers.